modules/ort/src/main/kotlin/workflow/analyzers/OrtResultAnalyzer.kt

```diff
@@ -37,6 +40,11 @@ class OrtResultAnalyzer : ManualAnalyzer() {
 
     override fun getName() = "OrtResult"
 
+    override fun configure(configMap: MutableMap<String, String>?) {
+        super.configure(configMap)
+        omitExcludedArtifacts = getBooleanConfigValue(omitExcludedArtifactsProperty, configMap)
```

Comment:
Hmm, it just occurred to me that `getBooleanConfigValue()` seem to hard-code the default value to `false`, so it seems we *cannot* have a default of `true` for the property, as this function always gets called (?) and therefore if does not matter which default value we assign to `omitExcludedArtifacts` above...

Reply:
We can probably fix this by doing:

    omitExcludedArtifacts = getConfigValue(omitExcludedArtifactsProperty, configMap, "true").toBoolean()

Reply:
See above